src/services/auth-service.js

```diff
@@ -0,0 +1,45 @@
+import axios from "axios";
+
+const loginHandler = async (loginCredentials, setAuthTokens, navigation) => {
+    try {
+        const { data } = await axios.post("/api/auth/login", {
+            email: loginCredentials.email,
+            password: loginCredentials.password,
+        });
+        const encodedToken = data.encodedToken;
+        localStorage.setItem("token", encodedToken);
+        setAuthTokens(encodedToken);
+        navigation("/");
+    } catch (error) {
+        console.log(error);
+    }
+};
```

Comment:
Use a separate file for the API calls.
Its good practice to move only the API calls into the service folder and then get that function into a different file